tests/python/test_arrays.py

```diff
@@ -1098,7 +1098,8 @@ class Counter {
     assert (False)
   except Exception as e:
     assert str(type(e)) == "<class 'pythonmonkey.SpiderMonkeyError'>"
-    assert str(e).__contains__("TypeError: this is null")
+    assert "TypeError:" in str(e)
```

Comment:
why is this better?

Reply:
`in` is a basic in Python syntax.
https://realpython.com/python-in-operator/#strings

Reply:
`__contains__` is the magic method for user-defined classes to implement the `in` operator, which shouldn't be used directly.

https://docs.python.org/3/reference/expressions.html#membership-test-details
> For the string and bytes types, x in y is True if and only if x is a substring of y. An equivalent test is y.find(x) != -1. Empty strings are always considered to be a substring of any other string, so "" in "abc" will return True.
> 
> For user-defined classes which define the [`__contains__()`](https://docs.python.org/3/reference/datamodel.html#object.__contains__) method, x in y returns True if `y.__contains__(x)` returns a true value, and False otherwise.